README.md

```diff
@@ -183,6 +183,21 @@ Pylance provides users with the ability to customize their Python language suppo
         -   `true`
         -   `false` (default)
 
+-    `python.analysis.inlayHints.pytestParameters`
+    -    Enable/disable inlay hints for pytest function parameters.
+    ```python
+    def test_foo(my_fixture):
+        assert(myfixture)
+    ```
+    becomes
+    ```python
+    def test_foo(my_fixture: str):
```

Comment:
Maybe you should include the fixture in the example code so the reader understands why `str` might make sense?